clang/include/clang/Basic/DiagnosticGroups.td

```diff
@@ -1387,3 +1387,6 @@ def BranchProtection : DiagGroup<"branch-protection">;
 // HLSL diagnostic groups
 // Warnings for HLSL Clang extensions
 def HLSLExtension : DiagGroup<"hlsl-extensions">;
+
+// Warnings about double precision arithmetic in device code
+def DoubleInDevice : DiagGroup<"double-in-device">;
```

Comment:
No need to add this as an explicit group, unless you intended to put the group inside of some other warning group?